### PR TITLE
Add ecto version detection for the changes in 3.5 to embed loading

### LINKED ISSUE #19: Loading embeds breaks with ecto 3.5
https://github.com/CargoSense/fable/blob/ae6b288045a8fbf5b5eae4c23228e897ffbb68f3/lib/fable/event.ex#L38-L43

https://github.com/elixir-ecto/ecto/commit/59962034a25835a40d15d6c7d8eae23e64fd4eba <- the commit changing how embeds are internally represented.

### PR DESCRIPTION
**Pull request by LostKobrakai**
Closes #19

### CODE REVIEW

File: lib/fable/event.ex [hunk 1]
```diff
@@ -43,15 +43,33 @@ defmodule Fable.Event do
...
        field: :data,
        on_cast: nil,
        on_replace: :raise,
        owner: __MODULE__,
        related: module,
        unique: true
-     }}
+     }
+     
+    if dependency_vsn_match?(:ecto, "~> 3.5") do
```
Comment (benwilson512) [inline lib/fable/event.ex]: This could be accomplished at compile time right?

Comment (LostKobrakai) [reply lib/fable/event.ex]: Ah, yeah. I mostly made sure it wasn't blowing up on my side :D

**Comment by benwilson512** [comment]
Thanks!

**Comment by LostKobrakai** [comment]
@benwilson512 would be great if you could add an `hacktoberfest-accepted` label here :)

**Comment by benwilson512** [comment]
Done!

**Pull request merged**